deploy/kubernetes/base/node_setup/linux/node.yaml

```diff
@@ -15,21 +15,20 @@ spec:
       # Host network must be used for interaction with Workload Identity in GKE
       # since it replaces GCE Metadata Server with GKE Metadata Server. Remove
       # this requirement when issue is resolved and before any exposure of
-      # metrics ports.
+      # metrics ports. But hostNetwork is not working for Windows, might be an issue
```

Comment:
Do we need the windows comment in the linux file?

Reply:
removed it.